sdk/include/ndk/mmfuncs.h

```diff
@@ -401,12 +401,12 @@ NTAPI
 ZwMapViewOfSection(
     _In_ HANDLE SectionHandle,
     _In_ HANDLE ProcessHandle,
-    _Outptr_result_bytebuffer_(*ViewSize) PVOID *BaseAddress,
```

Comment:
I think the original intent here was to say that the buffer pointed by the obtained base address, is of size given by the returned value pointed by ViewSize.

Reply:
> original intent

I assume so too.
Nonetheless, this parameter was and is stll an `_Inout_` (`*BaseAddress` is read too), not an `_Outptr_` (only wrtiiten).
Note:
If I remember correctly, such abusive changes were made elsewhere too.
But that will be a follow-up PR.

Reply:
Add `_Pre_valid_` to the existing annotation instead.

Reply:
> Add `_Pre_valid_`

Thanks for the hint.
Please, double-check, as I lack detailed SAL2 documentation.

Reply:
See
https://www.codeproject.com/Reference/879527/SAL-Function-Parameters-Annotations
https://learn.microsoft.com/en-us/cpp/code-quality/using-sal-annotations-to-reduce-c-cpp-code-defects?view=msvc-170

Reply:
Thanks. I know these helpful pages, but they don't say anything about `_Pre_valid_`, for example.